pkg/api/v1/pod/util.go

```diff
@@ -51,9 +51,7 @@ func FindPort(pod *v1.Pod, svcPort *v1.ServicePort) (int, error) {
 
 // TODO: remove this function when init containers becomes a stable feature
 func SetInitContainersAndStatuses(pod *v1.Pod) error {
-	var initContainersAnnotation string
-	initContainersAnnotation = pod.Annotations[v1.PodInitContainersAnnotationKey]
-	initContainersAnnotation = pod.Annotations[v1.PodInitContainersBetaAnnotationKey]
+	initContainersAnnotation := pod.Annotations[v1.PodInitContainersBetaAnnotationKey]
```

Comment:
please refer to `Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec` on how to handle this.